docs/developer/decision-records/2022-06-20-mvd-onboarding/README.md

```diff
@@ -0,0 +1,29 @@
+# Participants onboarding in MVD
+
+This document outlines the infrastructure steps for the scenario of including participants in the Minimum Viable Dataspace (MVD).
+
+## Precondition - GAIA-X membership
+
+To join the MVD, participants must have Verifiable Credentials signed by GAIA-X Authority which prove GAIA-X membership.
+
+At present the MVD deployment pipeline does not interact with the real GAIA-X Authority. A simulated GAIA-X Authority is deployed within MVD for demonstration purposes.
+
+## Onboarding process - GAIA-X membership verification
+
+During the onboarding process, the GAIA-X membership claim is verified by the Dataspace Authority.
+The onboarding flow is presented in the [distributed authorisation sub-flow document](https://github.com/agera-edc/MinimumViableDataspace/tree/feature/20-rs-adr-target/docs/developer/decision-records/2022-06-16-distributed-authorization).
```

Comment:
change to relative URL

Reply:
Yes, I already added a comment to track that

Reply:
https://github.com/agera-edc/MinimumViableDataspace/pull/153#discussion_r901879153

Reply:
changed